rpm/cvmfs-config-osg.spec

```diff
@@ -22,6 +22,8 @@ Provides: oasis-config = 9
 
 Conflicts: cvmfs-config-default
 
+Requires: cvmfs >= 2.6.0
```

Comment:
No.  As I said in the ticket, we won't be having a Requires in cvmfs-config-osg.  cvmfs and cvmfs-config-osg will continue to have "=" in osg-oasis.  Only cvmfs-x509-helper will be ">=" in osg-oasis.

Reply:
@DrDaveD what is the problem on having a version requirement on cvmfs-config-osg?

Reply:
It's just one more thing to maintain and doesn't get us anything beyond the setting in osg-oasis.  Even if somebody didn't have osg-oasis installed, I don't expect anybody to do an upgrade of cvmfs-config-osg without upgrading cvmfs.  In fact the newer cvmfs-config-osg does work with older cvmfs versions.  There is also a pending update to cvmfs-config-osg [SOFTWARE-3736](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3736) that I am holding off until cvmfs-2.6.1 which is expected any day.